### DISCUSSION TITLE
6.1 Adding PostGIS data base to Geoserver

### DISCUSSION THREAD

**Discussion by khartman-nj** [ua-gist604b-f24/syllabus#33] [General]
Hello I am not really sure what is going on here but I cant access the post gis data base in the geoserver I get the following error.
![issues](https://github.com/user-attachments/assets/5506a0d0-0b6c-4aed-a840-0b7122f468ba)
Any help is greatly apricated

**Comment by travisz09** [comment]
I and several other people also got stuck here. Robert (@robertgamboa10) pointed out that you need to change the host to postgis.

See [https://github.com/ua-gist604b-f24/syllabus/discussions/31](https://github.com/ua-gist604b-f24/syllabus/discussions/31).

**Reply by khartman-nj** [reply]
I tried that still got the same error and tried a different browser

**Reply by khartman-nj** [reply]
And still no dice it wont let me get around the security to access the data

**Reply by travisz09** [reply]
Did you set your database to hawaii? Also make sure to set the password to postgres, it is very annoying that the password keeps resetting every time you try to connect.
